walletsetup.go

```diff
@@ -373,6 +373,16 @@ func promptConsoleSeed(reader *bufio.Reader) ([]byte, error) {
 		}
 
 		seedStrTrimmed := strings.TrimSpace(seedStr)
+		wordCount := strings.Count(seedStrTrimmed, " ") + 1
+
+		if wordCount == 1 {
+			seed, err := hex.DecodeString(seedStrTrimmed)
```

Comment:
This will error if it has an odd number of characters.  That can be fixed by prepending a "0".  I believe there's a function to do this somewhere already but it's unexported and not sure if it's in the main package...